contrib/epee/include/net/connection_basic.hpp

```diff
@@ -135,7 +135,7 @@ class connection_basic { // not-templated base class for rapid developmet of som
 		bool handshake(boost::asio::ssl::stream_base::handshake_type type)
 		{
 			//m_state != nullptr verified in constructor
-			return m_state->ssl_options().handshake(socket_, type);
+			return !bool(m_state->ssl_options().handshake(socket_, type));
```

Comment:
Why ! ?

Reply:
`handshake` returns a `boost::system::error_code` which has inverted `bool` behavior unfortunately. Wrapping it in an `expect<void>` gives expected `bool` conversion, but that class would have to be moved to the `epee` playground. I could do `bool`+out parameter for `error_code` here ?

Reply:
No, as long as it's intended it's fine.